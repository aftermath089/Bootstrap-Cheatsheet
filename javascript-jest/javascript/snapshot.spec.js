const user = {
    name: 'john doe',
    age: '42',
    job: 'inventor'
}

test('match user snapshot', () =>{
    expect(user).toMatchSnapshot() //to save the snapshot using toMatchSnapshot(). if you want to update the snapshot file run the npm test with -u
})