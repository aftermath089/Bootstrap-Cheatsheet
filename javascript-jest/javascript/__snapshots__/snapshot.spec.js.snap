// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match user snapshot 1`] = `
Object {
  "age": "42",
  "job": "inventor",
  "name": "john doe",
}
`;
